exporter/otlphttpexporter/README.md

```diff
@@ -0,0 +1,35 @@
+# OTLP/HTTP Exporter
+
+Exports traces and/or metrics via HTTP using
+[OTLP](https://github.com/open-telemetry/opentelemetry-specification/blob/master/specification/protocol/otlp.md) format.
+
+The following settings are required:
+
+- `endpoint` (no default): The target URL to send data to (e.g.: https://example.com:55681/v1/traces).
+
+The following settings can be optionally configured:
+
+- `insecure` (default = false): when set to true disables verifying the server's
+  certificate chain and host name. The connection is still encrypted but server identity
+  is not verified.
+- `ca_file` path to the CA cert. For a client this verifies the server certificate. Should
+  only be used if `insecure` is set to false.
+- `cert_file` path to the TLS cert to use for TLS required connections. Should
+  only be used if `insecure` is set to false.
+- `key_file` path to the TLS key to use for TLS required connections. Should
+  only be used if `insecure` is set to false.
+- `timeout` (default = 30s): How long to wait until the connection is close.
```

Comment:
```suggestion
- `timeout` (default = 30s): How long to wait until the connection is closed.
```

Does this refer to keep-alive connections or the initial connection?  If the initial connection then normally this refers to how long to wait to establish the connection, so technically it cannot be closed if it is never established.

Reply:
This is actually incorrect description. Fixed now.